resources/views/discord/modal.blade.php

```diff
@@ -0,0 +1,38 @@
+<div id="discord-modal" class="modal fade" tabindex="-1" role="dialog">
+    <div class="modal-dialog">
+        <div class="modal-content">
+            <div class="modal-header">
+                <h4 class="modal-title">S.A. Proto Discord <i class="fab fa-discord ml-1 fa-flip-horizontal"></i></h4>
+                <button type="button" class="close" data-dismiss="modal" aria-label="Close">
+                    <span aria-hidden="true">&times;</span>
+                </button>
+            </div>
+            <div class="modal-body">
+
+                <p>
+                    <b>
+                        We have a Discord server!
+                    </b>
+                    Whenever you want to chill, join online activities or study together, you can join our Discord server.
+                </p>
+                <p>
+                    If this is your first time joining the Discord server, you'll have to get verified as a Proto member by the board.
+                    Send them a personal message on Discord, and you'll get verified. Don't forget to read the "Rules and Info"
+                    channel to be sure you know what you should and shouldn't do!
+                </p>
+
+            </div>
+            <div class="modal-footer">
+
+                <a href="https://discord.com/" class="btn btn-default pull-left" target="_blank">
+                    About Discord
+                </a>
+
+                <a href="https://invite.gg/proto" class="btn btn-success" target="_blank">
```

Comment:
Iewl hardcoded link. :(

Reply:
maar het is een perma link, wat zou ik er anders mee moeten doen, in een config stoppen?

Reply:
Environment variable or config would indeed be fine, just like our proto.utwente.nl domain name and the name of our secretary. :)

Reply:
Alright! 👍